### PR TITLE
Adding tip about jQuery UJS

### PR DESCRIPTION
**Pull request by andreynering**
Reference #18533

### CODE REVIEW

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -1485,6 +1485,9 @@ Without this file, the confirmation dialog box wouldn't appear.
 
 ![Confirm Dialog](images/getting_started/confirm_dialog.png)
 
+TIP: Learn more about jQuery Unobstrusive Adapter (jQuery UJS) on
```
Comment (georgeclaghorn) [inline guides/source/getting_started.md]: Typo? Should be "Unobtrusive."

Comment (andreynering) [inline guides/source/getting_started.md]: @georgeclaghorn Yes, it was a typo. Fixed.

File: guides/source/getting_started.md [hunk 2]
```diff
@@ -1485,6 +1485,9 @@ Without this file, the confirmation dialog box wouldn't appear.
 
 ![Confirm Dialog](images/getting_started/confirm_dialog.png)
 
+TIP: Learn more about jQuery Unobtrusive Adapter (jQuery UJS) on
```
Comment (seuros) [inline guides/source/getting_started.md]: Could you change TIP to Note ?

Comment (andreynering) [inline guides/source/getting_started.md]: @seuros Why? All the others references to another Rails guide are tips, not notes.

Comment (seuros) [inline guides/source/getting_started.md]: I didn't notice that. :+1:

**Comment by fera2k** [comment]
Thanks Andrey! Good tip for the begginers like myself.

**Pull request merged**